packages/suite/src/utils/wallet/coinjoinUtils.ts

```diff
@@ -58,27 +58,21 @@ export const breakdownCoinjoinBalance = ({
 
 /**
  * Transform from coordinator format to coinjoinReducer format `CoinjoinClientFeeRatesMedians`
- * array => object { name: value-in-vBytes }
+ * array => object { name: value-in-kvBytes }
  */
-export const transformFeeRatesMedians = (medians: CoinjoinStatusEvent['feeRatesMedians']) => {
-    const [fast, recommended, slow] = medians.map(m => m.medianFeeRate);
-    // convert from kvBytes (kilo virtual bytes) to vBytes
-    const kvB2vB = (v: number) => (v ? Math.round(v / 1000) : 1);
-    return {
-        fast: kvB2vB(fast) * 2, // NOTE: this calculation will be smarter once have enough data
-        recommended: kvB2vB(recommended),
-        slow: kvB2vB(slow),
-    };
-};
+export const transformFeeRatesMedians = (medians: CoinjoinStatusEvent['feeRatesMedians']) => ({
```

Comment:
i think i was better before (tranformed data in reducer), you dont need to perform calculations on each render of `CoinjoinDefaultStrategy` component (kvB2vB inside) or any other component which will potentially require those data 

btw this was [your argument ](https://github.com/trezor/trezor-suite/pull/6563#issuecomment-1279027774) and the reason why i've done it :) did you change your mind?

calculation (multiplication) will be done only when calling `actions.startCoinjoinSession` function, you already doing it when custom feeRate

Reply:
Yeas, I am torn. Changing it back. I did not like how it gets transformed there and back again, but it seems like a lesser evil. 93673b2e59367ed13545dc2e5117e9c1433d505d